Cargo.toml

```diff
@@ -33,7 +33,7 @@ exclude = [
 ]
 
 [features]
-default = ["compat-3-0-0"]
+default = ["compat-3-0-0", "chrono-0_4", "serde_with-3", "serde"]
```

Comment:
Let's keep these out of the default features list. Were you having an issue testing without these enabled here?